vgaidarji/github-actions-pr-merger
# Add npm lint command and reorganize folders accordingly

### What has been done
1. Added `npm lint` command and reorganized folders accordingly

### How to test
1. Run `npm run lint` and make sure only `.js` files from the project are being verified
